### PR TITLE
feat: added cards

### PR DESCRIPTION
**Pull request by Tanjum786**
**Added cards to the component**
I have created 5 variants of cards as below:

- Basic card with shadow effect
- Horizontal card
- overlay card 
- DIsmiss card 
- Badge on card

**Please comment your valuable reviews**  

Screenshot from 2022-02-12 01-47-05
Screenshot from 2022-02-12 01-47-19
Screenshot from 2022-02-12 01-47-22

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -702,51 +699,227 @@ <h1 class="component_heading">Font family</h1>
...
-
-
-
-
-
-
-
+            <h2 class="component_example">Example of Card with Dismiss</h2>
+            <div class="component_box">
+                <!-- card with dismiss -->
```
Comment (vaibhav-mahalle) [inline index.html]: no need to add the comment here. There is heading for the particular card.

File: index.html [hunk 2]
```diff
@@ -702,51 +699,227 @@ <h1 class="component_heading">Font family</h1>
...
+                    style="width: 1203px; height: 745px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+            </div>
+            <h2 class="component_example">Example of Card with Badge</h2>
+            <div class="component_box">
+                <!-- card with badge -->
```
Comment (vaibhav-mahalle) [inline index.html]: same as above

File: stylestheet/component/card.css [hunk 3]
```diff
@@ -1,65 +1,153 @@
...
-}
-i:hover{
-    color:#3b82f6;
 }
+.card_container {
+    background-color: white;
+    box-shadow: 3px 2px 4px 2px gray;
+}
+
+.card_imge {
```
Comment (vaibhav-mahalle) [inline stylestheet/component/card.css]: either you can use card_img or card_image this looks little odd. just a minor suggestion.

File: stylestheet/component/card.css [hunk 4]
```diff
@@ -1,65 +1,153 @@
...
+    height: 16rem;
+    border-top-left-radius: 5px;
+    border-top-right-radius: 5px;
+}
+
+.card_horizental,
```
Comment (vaibhav-mahalle) [inline stylestheet/component/card.css]: you can change it to card-horizontal

File: stylestheet/component/card.css [hunk 5]
```diff
@@ -1,65 +1,153 @@
...
+    font-size: 3rem;
+    left: 5rem;
+    top: 18rem;
+}
+
+.card_opacity {
+    opacity: 0.4;
+}
+
+/* horizental card */
```
Comment (vaibhav-mahalle) [inline stylestheet/component/card.css]: same as above.

**Review by vaibhav-mahalle — APPROVED** [review]
suggested some changes. code structure is good just look into component naming a little bit.

File: stylestheet/component/card.css (hunk 3 above)
Comment (Tanjum786) [reply stylestheet/component/card.css]: ok i will change

**Review by RahulSingh9131 — APPROVED** [review]
cards look good to me. You can place your badge near the top of the card container, that would make it look even more better.

**Pull request merged**